lua/entities/terminator_nextbot_zambiemecha/shared.lua

```diff
@@ -0,0 +1,248 @@
+AddCSLuaFile()
+
+ENT.Base = "terminator_nextbot_zambie"
+DEFINE_BASECLASS( ENT.Base )
+ENT.PrintName = "Mecha Zombie"
+ENT.Spawnable = false
+ENT.Author = "regunkyle"
+list.Set( "NPC", "terminator_nextbot_zambiemecha", {
+    Name = "Mecha Zombie",
+    Class = "terminator_nextbot_zambiemecha",
+    Category = "Nextbot Zambies",
+} )
+
+ENT.SpawnHealth = 1500
+ENT.WalkSpeed = 70
+ENT.MoveSpeed = 180
+ENT.RunSpeed = 300
+ENT.AccelerationSpeed = 1000
+ENT.FistDamageMul = 1.5
+ENT.MyPhysicsMass = 200
+ENT.JumpHeight = 200
+
+ENT.DoMetallicDamage = true
+ENT.MetallicMoveSounds = true
+ENT.ReallyStrong = true
+ENT.ReallyHeavy = true
+
+ENT.HasBrains = true
+ENT.IsStupid = false
+ENT.IsFodder = false
+
+ENT.term_SoundPitchShift = -30
+ENT.term_SoundLevelShift = 10
+
+ENT.Mecha_LastShockwave = 0
+ENT.Mecha_ShockwaveCooldown = 5
+
+ENT.TERM_MODELSCALE = 1.5

```

Comment:
This feels pretty big when spawned in, 1.5 is definitely more of an "elite" size